packages/eslint-config-airbnb-base/package.json

```diff
@@ -57,7 +57,7 @@
     "babel-preset-airbnb": "^4.0.1",
     "babel-tape-runner": "^3.0.0",
     "eclint": "^2.8.1",
-    "eslint": "^5.16.0 || ^6.1.0",
+    "eslint": "^5.16.0 || ^6.4.0",
```

Comment:
this change would need to be made in both dev deps and peerDeps; it also is arguably a breaking change, but I've typically treated this as semver-minor in the past.

Reply:
@ljharb Done, thanks for reviewing!